ngraph_bridge/ie_executable.cc

```diff
@@ -141,5 +154,36 @@ bool IE_Executable::call(const vector<shared_ptr<runtime::Tensor>>& outputs,
   m_infer_req.Infer();
   return true;
 }
+
+bool IE_Executable::call_trivial(
+    const vector<shared_ptr<runtime::Tensor>>& outputs,
+    const vector<shared_ptr<runtime::Tensor>>& inputs) {
+  // Do basic sanity checking first
+  if (!outputs.size()) {
+    THROW_IE_EXCEPTION << "Called trivial IE function with no outputs";
+  }
+  if (outputs.size() && outputs.size() != 1) {
```

Comment:
Shouldn't outputs.size() != 1 be enough?

Reply:
Yes, fixed.